lib/codeowners/cli/helpers/suggest_subgroup_for_pattern.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+require 'forwardable'
+
+module Codeowners
+  module Cli
+    module Helpers
+      # This class covers 'suggestion of subgroups for pattern' process:
+      #
+      # - If there are any existing subgroups which are belong to selected new file owner, then
+      #   it shows 'suggestion dialog' with ordered list of subgroups to select.
+      #
+      #   EG:
+      #
+      #     Owners:
+      #     1 - @toptal/bootcamp
```

Comment:
Let's use fake names here and skip `toptal groups`. Like `@company/backend`, `@company/frontend` etc.